cmd/loopd/daemon.go

```diff
@@ -7,34 +7,71 @@ import (
 	"net/http"
 	"os"
 	"os/signal"
+	"path/filepath"
 	"runtime/pprof"
 	"sync"
 	"time"
 
 	proxy "github.com/grpc-ecosystem/grpc-gateway/runtime"
 	"github.com/lightninglabs/loop"
+	"github.com/lightninglabs/loop/lndclient"
 	"github.com/lightninglabs/loop/looprpc"
 	"google.golang.org/grpc"
 )
 
 // daemon runs loopd in daemon mode. It will listen for grpc connections,
 // execute commands and pass back swap status information.
 func daemon(config *config) error {
+	// Based on the network, we may need to swap our items like the
+	// swapserver or the default location of macaroons if a path isn't set.
+	switch config.Network {
+	case "testnet":
+		// If the user is targeting the testnet network, then we'll
+		// point towards the testnet swap server rather than the
+		// mainnet endpoint.
+		config.SwapServer = testnetServer
+
+		if config.Lnd.MacaroonPath == "" {
```

Comment:
Rename config option to `MacaroonDir`? It isn't the option that it used to be. If we want to keep using `MacaroonPath`, the error further down referencing "macaroon dir" should be updated.

Reply:
Renamed!